test/manual/api/system_api_legacy_tests.sh

```
@@ -0,0 +1,531 @@
+      data_array="$response_body"
+    fi
+    
+    # Check each created system
+    local all_systems_in_list=true
+    for sid in $CREATED_SYSTEM_IDS; do
+      if echo "$data_array" | jq -e ".[] | select(.solar_system_id == $sid)" >/dev/null 2>&1; then
+        if [ "${QUIET_MODE:-0}" -ne 1 ]; then
+          echo "✓ System $sid appears in list API after creation"
+        fi
+      else
+        all_systems_in_list=false
```

Shellcheck (0.10.0)

[warning] all_systems_in_list appears unused. Verify use (or export if used externally).

(SC2034)